tests/gitlab_api_mock.py

```diff
@@ -177,25 +175,27 @@ def add_merge_request(self, info, sudo=None, from_state=None, to_state=None):
         self.add_resource('/projects/{0.project_id}/merge_requests/{0.iid}', info, sudo, from_state, to_state)
         self.add_transition(
             GET(
-                '/projects/{0.project_id}/merge_requests/{0.iid}'.format(attrs(info)),
+                f'/projects/{attrs(info).project_id}/merge_requests/{attrs(info).iid}',
                 args={'include_rebase_in_progress': 'true'},
             ),
             Ok(info),
             sudo, from_state, to_state,
         )
 
     def add_commit(self, project_id, info, sudo=None, from_state=None, to_state=None):
+        # pylint: disable=consider-using-f-string
         path = '/projects/%s/repository/commits/{0.id}' % project_id
         self.add_resource(path, info, sudo, from_state, to_state)
```

Comment:
The reason for disabling the check here is that `add_resource` formats the input too using `.format(..)`.